qa/mc_sc_forging4.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python2
+
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration
+from test_framework.util import initialize_chain_clean, start_nodes, \
+    websocket_port_by_mc_node_index
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, start_sc_nodes, generate_next_blocks
+from SidechainTestFramework.sc_forging_util import *
+
+
+"""
+Check Latus forger behavior for:
+1. Sidechain has multiple MC blocks to be synchronized (>50). Check that sidechains forging not fails due to 
+   time out(SC create too many requests to MC for block generation).
+
+Configuration:
+    Start 1 MC nodes and 1 SC node.
+    SC node connected to the first MC node.
+
+Test:
+    - Synchronize MC nodes to the point of SC Creation Block.
+    - Mine 50 MC blocks, then forge SC block, verify MC data inclusion
+    - Mine 150 MC blocks, then forge 2 SC blocks, verify MC data inclusion
+    - Mine 200 MC blocks, then forge SC block, verify MC data inclusion  
```

Comment:
Let's simplify the test flow:
- Mine 200 MC blocks
- Forger N SC blocks and verify MC block headers  and ref data inclusion.
Keeping in mind max number of 49 MC headers per SC block for now.

Later we will update the test case according to the changes in Forger logic.